linters/kube-linter/plugin.yaml

```diff
@@ -28,5 +13,5 @@ lint:
       environment:
         - name: PATH
           list: ["${linter}"]
-      known_good_version: 0.5.0
+      known_good_version: 0.6.4
```

Comment:
Please rename snapshots too. There should always be a snapshot for the `known_good_version` value (this wouldn't cause tests to fail, but it's a repo cleanliness thing and makes it easier to keep track of what is running what)